README.md

```diff
@@ -408,12 +415,15 @@ As such, it's recommended to use `stream_set_read_buffer($stream, 0);`
 to disable PHP's internal read buffer in this case.
 See also [`addReadStream()`](#addreadstream) for more details.
 
-#### ExtLibevLoop
+#### ~~ExtLibevLoop~~
```

Comment:
LGTM, but make sure to also update the TOC similar to #231 :+1:

Reply:
Done 👍